lib/shared/messages.js

```diff
@@ -52,6 +52,12 @@ module.exports = {
       'You are currently flashing a drive. Closing Etcher may leave',
       'your drive in an unusable state.\n\n',
       'Are you sure you want to close Etcher?'
+    ].join(' ')),
+
+    possiblyWindowsImage: _.template([
+      'It looks like you are trying to write a Windows image.',
+      'Windows images require a special process to be written as bootable images.',
+      'Please check the Etcher documentation for more info.'
```

Comment:
I wonder if this message should somehow clarify that Etcher itself isn't capable of doing this "special process"?

(the current message could be mis-interpreted that the Etcher documentation contains info on how to get Etcher to do the magic steps?)